assets/json-schema-faker.js

```diff
@@ -24232,139 +24232,257 @@ function extend() {
   // fallback generator
   var anyType = { type: ['string', 'number', 'integer', 'boolean'] };
   // TODO provide types
+  // Updated objectType definition to latest version (0.5.0-rcv.41)
```

Comment:
We can avoid a detailed review of this part. It's replaced by latest code present over here: https://github.com/json-schema-faker/json-schema-faker/blob/master/src/lib/types/object.js